src/monikaArkade/Test1_nonRepeatingCharacters.java

```diff
@@ -0,0 +1,29 @@
+package monikaArkade;
+
+public class Test1_nonRepeatingCharacters {
+	int getcharfreq(String name ,char ch) {
+		int count =0;
+		for(int index=0;index<name.length();index++) {
+			if(name.charAt(index) == ch)
+				count++;		
+		}
+		return count;
+	}
+
+	void findNonReChar(String name) {
+		char chr =' ';
+		for(int index=0;index<name.length();index++) {
+			char ch =name.charAt(index);
+			int count =getcharfreq(name ,ch);
+			if(count == 1) {
```

Comment:
here you can aslo use 
indexOf(ch) ==lastIndexOf(ch) logic to avoid one more for loop